poetry.lock

```diff
@@ -0,0 +1,672 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
This was apparently generated by poetry < 1.4.0 (the current) release. Maybe update your local poetry installation (`poetry self update`) and recreate lock file to get the most up-to-date result.